SPECS/pango/pango.spec

```diff
@@ -1,58 +1,68 @@
-Summary:	library for laying out and rendering of text.
-Name:		pango
-Version:	1.40.4
-Release:    4%{?dist}
-License:	LGPLv2 or MPLv1.1
-URL:		http://pango.org
-Group:		System Environment/Libraries
+Summary:        library for laying out and rendering of text.
+Name:           pango
+Version:        1.44.7
+Release:        1%{?dist}
+License:        LGPLv2 OR MPLv1.1
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	    https://download.gnome.org/sources/pango/1.40/%{name}-%{version}.tar.xz
-BuildRequires:	glib-devel
-BuildRequires:	cairo
-BuildRequires:	cairo-devel
-BuildRequires:	libpng-devel
-BuildRequires:	fontconfig
-BuildRequires:	fontconfig-devel
-BuildRequires:	harfbuzz
-BuildRequires:	harfbuzz-devel
-BuildRequires:	freetype
-Requires:	    harfbuzz-devel
+Group:          System Environment/Libraries
+URL:            https://pango.org
+Source0:        https://download.gnome.org/sources/pango/1.44/%{name}-%{version}.tar.xz
+BuildRequires:  cairo
```

Comment:
Can we drop the base packages? It seems that `-devel` packages depend on the base ones already.